packages/crawler/tests/e2e/specs/index.e2e.ts-snapshots/should-generate-raw-data-correctly-for-1-version-and-2-prefixes-1-chrome-linux.txt

```diff
@@ -0,0 +1 @@
+{"latest":{"my-prefix":[{"p-text":{"properties":{"tag":"p","size":"small","weight":"regular","align":"start","color":"primary","ellipsis":false,"theme":"light"},"children":"First prefix","hostPdsComponent":null}}],"":[{"p-text":{"properties":{"tag":"p","size":"small","weight":"regular","align":"start","color":"primary","ellipsis":false,"theme":"light"},"children":"Second prefix","hostPdsComponent":null}}]}}
```

Comment:
Would it be possible to prettify the result before creating the snapshot? Otherwise it might be hard to see the diff in case it differs in the future.

Reply:
e.g. `import { format } from 'prettier'; … await format(JSON.stringify(stats), { parser: 'json' })`